src/config.rs

```diff
@@ -32,29 +83,59 @@ pub(crate) struct FarmerConfig {
     pub(crate) plot_directory: PathBuf,
     #[serde(with = "bytesize_serde")]
     pub(crate) plot_size: ByteSize,
+    #[serde(default)]
     pub(crate) opencl: bool,
```

Comment:
There is no mention of OpenCL in Subspace monorepo anymore, I think this needs to go away.

Reply:
will do